sdk/core/Azure.Core/src/Pipeline/Policies/LoggingPolicy.cs

```diff
@@ -13,7 +13,7 @@ namespace Azure.Core.Pipeline.Policies
 {
     public class LoggingPolicy : HttpPipelinePolicy
     {
-        private static readonly long s_delayWarningThreshold = 3000; // 3000ms
+        private const long s_delayWarningThreshold = 3000; // 3000ms
```

Comment:
Nit - if you have to update this PR, maybe change this from `s_delayWarningThreshold` to `DelayWarningThreshold`